src/Microsoft.DotNet.Build.Tasks/VersionTools/BaseDependenciesTask.cs

```diff
@@ -298,6 +319,11 @@ private FileRegexUpdater ConfigureFileRegexUpdater(FileRegexUpdater updater, ITa
                 updater.Regex = new Regex(manualRegex);
                 updater.VersionGroupName = GetRequiredMetadata(step, "VersionGroupName");
             }
+            else if (!string.IsNullOrEmpty(msbuildSdkName))
+            {
+                updater.Regex = CreateMSBuildSdkUpdateRegex(Regex.Escape(msbuildSdkName), nameof(msbuildSdkName));
```

Comment:
Escaping since the name will almost always have `.`, which needs to become `\.`

MSBuildSdkName and PackageId will also always be the same, but there isn't an easy/existing way to switch on the different types of updaters right now

Reply:
> there isn't an easy/existing way to switch on the different types of updaters right now

Yeah, I can imagine it being better with more general refactoring, but we have no automated tests (and manual tests are tedious without some extra scripts on top). Looks good for now.